routes/api.client.php

```diff
@@ -17,13 +17,12 @@
     return response()->json([
         'version' => GetCandy::version(),
         'locale' => app()->getLocale(),
-        'channel' => new GetCandy\Api\Http\Resources\Channels\ChannelResource($channel->getChannel()),
+        'channel' => new \GetCandy\Api\Core\Channels\Resources\ChannelResource($channel->getChannel()),
```

Comment:
We seem to be inconsistent with the backslashes at the start of namespaces, e.g. some are

`new GetCandy`

others

`new \GetCandy`

Can we keep to one?

Reply:
I've changed this now to use an action instead. Was thinking of even moving this logic out into the GetCandy facade, but not really part of this development...